articles/django-rest-api-mongodb-notes-app/index.md

```diff
@@ -0,0 +1,231 @@
+# Build a Notes App API with Django REST Framework and MongoDB
+
+## Introduction
+
+MongoDB is a NoSQL database system. It stores data in JSON-like format instead of the traditional row-column format. The Django REST framework is a powerful toolkit for building extensible APIs. The Django framework is a Python framework for web development.
+
+You will learn how to use the powerful Django REST framework and MongoDB in your web projects. You will learn how to put in place CRUD features in a notes app.
+
+In this post, you will:
+
+- Understand to integrate MongoDB into Django projects
+- Learn how to set up and run Django REST API
+- Create CRUD features in Django REST API
+- Build a Notes app API
+
+## Prerequisites
+
+- Python 3 installed on your machine. You can install Python 3 from the [Python website](https://www.python.org/download/releases/3.0/).
+- Experience with Python and Django
+
+## Setting up MongoDB Server
+
+You may download and install the MongoDB database server from the [official MongoDB website](https://www.mongodb.com/try/download/community).
+
```

Comment:
Setting up MongoDB is not always easy for beginners, or Linux users. Provide a link to a tutorial(s) to guide the installations.

Also add the instructions for creating the database for use with this tutorial.